_data/providers.json

```diff
@@ -400,6 +400,11 @@
       "url": "https://www.volcanohosting.net/",
       "description_template": "default"
     },
+    {
+      "name": "VoltHosting",
+      "url": "https://www.volthosting.co.uk/",
+      "description_template": "Enable clone-remote-port (or manually set bedrock port to the Java port) so that the Bedrock's port is the Java server’s port, and connect with that port. Alternatively, request an additional port, or buy a dedicated IP address.<br>[Contact us](https://volthosting.co.uk/contact.html) if you need further assistance."
```

Comment:
Please change `description_template` to just `description`, otherwise the description won't show properly. Additionally, "contact us" is a bit misleading since the info would be shown on Geysers wiki - please change that (see the the Apex/BisectHosting articles for reference on that)